src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java

```diff
@@ -5878,7 +5878,10 @@ public void testListPayments_withParameters() throws Exception {
         HyperwalletList<HyperwalletPayment> resp = client.listPayments(options);
         assertThat(resp, is(equalTo(response)));
 
-        Mockito.verify(mockApiClient).get(Mockito.eq("https://api.sandbox.hyperwallet.com/rest/v3/payments?createdAfter=2016-06-29T17:58:26Z&createdBefore=2016-06-29T17:58:26Z&sortBy=test-sort-by&offset=5&limit=10"), Mockito.any(TypeReference.class));
+        Mockito.verify(mockApiClient).get(Mockito
+                        .eq("https://api.sandbox.hyperwallet.com/rest/v3/payments?createdAfter=2016-06-29T17:58:26Z&createdBefore=2016-06-29T17:58"
+                                + ":26Z&sortBy=test-sort-by&offset=5&limit=10&clientPaymentId=gv47LDuf"),
```

Comment:
Nothing wrong here but usually, clientPaymentId filter should come before sort, offset and limit, will adding the .clientPaymentId("gv47LDuf") after limit(10) fix it?

Reply:
I don't think it will fix it. Existing code has pagination parameters put before actual field params like clientPaymentId. Check listPayments method in Hyperwallet.java.

Reply:
We should add this to pagination?